src/components/charts/pie-chart/pie-chart.js

```diff
@@ -0,0 +1,60 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import TooltipChart from 'components/charts/tooltip-chart';
+import {
+  ResponsiveContainer,
+  PieChart as RechartsPieChart,
+  Pie,
+  Tooltip
+} from 'recharts';
+
+class PieChart extends PureComponent {
+  render() {
+    const { config, data, height, margin, customTooltip } = this.props;
+    return (
+      <div>
+        <ResponsiveContainer height={height} margin={margin}>
+          <RechartsPieChart>
+            <Tooltip
+              isAnimationActive={false}
+              content={content =>
+                customTooltip &&
+                  React.cloneElement(customTooltip, { content, config }) ||
+                  <TooltipChart content={content} config={config} />}
+              filterNull={false}
+            />
+            <Pie
+              data={data}
+              dataKey="value"
+              fill={config.theme && config.theme.fill}
```

Comment:
Maybe is a bit misleading that the `Pie` `fill` property is not used and instead the `data` `fill` key is the one used. I understand this is how Recharts works under the hood, but wouldn't it be clearer to be more explicit?:
```
              <Pie
                data={data} 
                dataKey="value"
                fill={config.theme && config.theme.fill}
              >
        	    {
          	    data.map((d, index) => <Cell fill={d.fill}/>)
               }
             </Pie>
```